packages/Frontend/src/components/SignIn.jsx

```diff
@@ -1,19 +1,40 @@
-import React, { Fragment } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
+import { Button, Divider, Form, Header, Icon, Loader, Segment, Message } from 'semantic-ui-react';
 
 export default function SignIn({
   error, loading, signIn, submitIfEnter, updateFormByKey,
 }) {
   return (
-    <Fragment>
-      <p>Username: </p>
-      <input onChange={updateFormByKey('username')} onKeyUp={submitIfEnter} />
-      <p>Password: </p>
-      <input onChange={updateFormByKey('password')} onKeyUp={submitIfEnter} type="password" />
-      <button onClick={signIn}>Submit</button>
-      { loading ? <p>Loading...</p> : null }
-      { error ? <p>{error}</p> : null }
-    </Fragment>
+    <div style={{
+        margin: 'auto',
+        position: 'relative',
+        top: 100,
+        width: '50%',
+      }}
+    >
+      <Segment>
+        <Header as="h1" icon textAlign="center">
+          <Icon name="user" circular />
+          <Header.Content>
+            Sign In
+          </Header.Content>
+        </Header>
+        <Form>
+          <Form.Input placeholder="Username" onChange={updateFormByKey('username')} onKeyUp={submitIfEnter} />
+          <Form.Input placeholder="Password" onChange={updateFormByKey('password')} onKeyUp={submitIfEnter} type="password" />
```

Comment:
FYI, updateFormByKey is another curried function here, it allows both forms to use mostly the same code, but set two different keys in state.